nav2_amcl/include/nav2_amcl/amcl_node.hpp

```diff
@@ -137,6 +137,8 @@ class AmclNode : public nav2_lifecycle::LifecycleNode
     double & x, double & y, double & yaw,
     const rclcpp::Time & sensor_timestamp, const std::string & frame_id);
   std::atomic<bool> first_pose_sent_;
+  std::atomic<bool> amcl_node_ready_;
+  std::atomic<bool> first_laser_scan_rec_;
```

Comment:
Where/how is this used? I don't see it being set or used in the code anywhere

Reply:
`first_laser_scan_rec_` should be removed.